src/content/reference/react/cacheSignal.md

```diff
@@ -0,0 +1,121 @@
+---
+title: cacheSignal
+---
+
+<Canary>
+
+**The `cacheSignal()` API is currently only available in React’s Canary and Experimental channels.** 
+
+[Learn more about React’s release channels here.](/community/versioning-policy#all-release-channels)
+
+</Canary>
+
+<RSC>
+
+`cacheSignal` is currently only used with [React Server Components](/blog/2023/03/22/react-labs-what-we-have-been-working-on-march-2023#react-server-components). 
+
+</RSC>
+
+<Intro>
+
+`cacheSignal()` returns an [`AbortSignal`](https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal) which aborts when the cache lifetime is done based on the same execution scope as a [`cache()`](/reference/react/cache)ed function 
```

Comment:
This needs to be explained simple enough to fit on one line in desktop. Something like:

```
`cacheSignal` allows you to know when a `cache()` is cleared.
```